docs/integrations/python.md

```diff
@@ -13,15 +13,18 @@ redirect_from:
 {% include toc_2-3.html %}
 
 
-**Improved Python SDK**  <span class="badge mr-1">New</span><span class="badge">beta</span>  
-Check-out the newly released [Python SDK library](https://pydocs-lakefs.lakefs.io/) - providing simpler interface and improved user experience!  
-*OR* continue reading for the current Python SDK.
+**High Level Python SDK**  <span class="badge mr-1">New</span>  
+We've just released a new High Level Python SDK library, and we're super excited to tell you about it! Continue reading to get the
+full story!  
+**For previous Python SDKs follow these links:**  
+[lakefs-sdk](https://pydocs-sdk.lakefs.io)  
+[legacy-sdk](https://pydocs.lakefs.io)
```

Comment:
Need to understand that the python sdk is still supported and used by the high level one, don't want people think we deprecate it as we did for the legacy.

Maybe my issue is the two primary ways we describe - as this package uses the sdk itself.

Reply:
> and used by the high level one

It's an implementation detail that the user shouldn't care about